frontstage/templates/help/help-who-is-ons.html

```diff
@@ -39,6 +39,11 @@ <h1 class="ons-u-fs-xl">Who is the Office for National Statistics (ONS)?</h1>
         <path d="M8.83,7.88a.51.51,0,0,0,.71,0l2.31-2.32,1.28,1.28A.51.51,0,0,0,14,6.49v-4a.52.52,0,0,0-.5-.5h-4A.51.51,0,0,0,9,2.52a.58.58,0,0,0,.14.33l1.28,1.28L8.12,6.46a.51.51,0,0,0,0,.71Z" transform="translate(-2 -1.99)" />
       </svg></a></p>
     <h2>Need more information?</h2>
-    <p><a href="{{ url_for('sign_in_bp.login') }}" rel="noopener">Sign in to your account</a> to send a secure message.</p>
-    <p>If you are having problems signing in, please <a href="{{ url_for('contact_us_bp.contact_us') }}" rel="noopener">contact us</a>.</p>
+    {% if authorization %}
```

Comment:
I'm not sure how an unauthorised user would get to this page now?

Reply:
I found a few ways to hit the endpoint when signed out. You can copy the url before signing out and then pasting the url back in to pick up after the sign out. It's also possible, that a user has signed out but remained on the message screen and then refreshes the screen.

Reply:
It is a bit weird, but current behaviour required